### PR TITLE
Eslint

### PR DESCRIPTION
**Pull request by markumed1**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,8 +21,14 @@
   "eslintConfig": {
     "extends": [
       "react-app",
-      "react-app/jest"
-    ]
+      "react-app/jest",
```
Comment (markumed1) [inline package.json:24]: Hi

Comment (markumed1) [inline package.json:24]: Jest is a JavaScript test runner that provides resources for writing and running tests.

File: package.json [hunk 2]
```diff
@@ -21,8 +21,14 @@
...
+      "eslint:recommended",
```
Comment (markumed1) [inline package.json:25]: A rule in eslint

File: package.json [hunk 3]
```diff
@@ -21,8 +21,14 @@
...
+      "plugin:react/recommended",
```
Comment (markumed1) [inline package.json:26]: An extension to ESLint which can perform numerous functions (? I believe this is the answer?)

File: package.json [hunk 4]
```diff
@@ -21,8 +21,14 @@
...
+      "plugin:prettier/recommended"
+    ],
+    "rules": {
+      "prettier/prettier": ["error", {"endOfLine": "auto"}]
```
Comment (markumed1) [inline package.json:30]: Same as above?

File: package.json [hunk 5]
```diff
@@ -21,8 +21,14 @@
...
+    }
   },
   "browserslist": {
```
Comment (markumed1) [inline package.json:33]: tool that allows specifying which browsers should be supported in your frontend app by specifying "queries" in a config file

File: package.json [hunk 6]
```diff
@@ -21,8 +21,14 @@
...
     "production": [
```
Comment (markumed1) [inline package.json:34]: minifies your code, optimizes assets, and produces lighter weight source maps.

File: package.json [hunk 7]
```diff
@@ -38,7 +44,10 @@
   },
   "devDependencies": {
```
Comment (markumed1) [inline package.json:45]: are packages used for development purposes, e.g for running tests or transpiring your code.

File: package.json [hunk 8]
```diff
@@ -38,7 +44,10 @@
...
     "autoprefixer": "^10.4.7",
+    "eslint-config-prettier": "^8.6.0",
+    "eslint-plugin-prettier": "^4.2.1",
     "postcss": "^8.4.14",
+    "prettier": "^2.8.3",
     "tailwindcss": "^3.1.6"
```
Comment (markumed1) [inline package.json:51]: version of tailwind

File: package.json [hunk 9]
```diff
@@ -38,7 +44,10 @@
   },
   "devDependencies": {
     "autoprefixer": "^10.4.7",
+    "eslint-config-prettier": "^8.6.0",
+    "eslint-plugin-prettier": "^4.2.1",
     "postcss": "^8.4.14",
+    "prettier": "^2.8.3",
```
Comment (markumed1) [inline package.json:50]: version of prettier program

**Review by kenttk — APPROVED** [review]

**Pull request merged**